### DISCUSSION TITLE
When Pi4J will be available for Raspberry Pi 5?

### DISCUSSION THREAD

**Discussion by piggy1130** [Pi4J/pi4j#323] [General]
I want to use the Pi4J library to read/send information through the GPIO in "**Raspberry Pi 5**". However, I realized that the current Pi4J library does not work for Raspberry Pi 5 :( Do you have any idea when Pi4J will be available for use with "**Raspberry Pi 5**"? 

Thank you so much for any response :)

**Comment by FDelporte** [comment]
See issue https://github.com/Pi4J/pi4j-v2/issues/321
